docs/notifications.json

```diff
@@ -13,5 +13,10 @@
     "id": 78,
     "title": "<b>MUI X v6.18.x and the latest improvements before the next major</b>",
     "text": "New stable components, polished features, better performance and more. Check out the details in our <a style=\"color: inherit;\" data-ga-event-category=\"Announcement\" data-ga-event-action=\"notification\" data-ga-event-label=\"mui-x-end-v6\" href=\"https://mui.com/blog/mui-x-end-v6-features/\">recent blog post</a>."
+  },
+  {
+    "id": 79,
+    "title": "<b>The MUI Developer Survey 2023 is live</b>",
+    "text": "<a style=\"color: inherit;\" data-ga-event-category=\"Announcement\" data-ga-event-action=\"notification\" data-ga-event-label=\"mui-survey\" href=\"https://tally.so/r/3Ex4PN?source=docs-notification\">Participate</a> and help shape MUI's roadmap for 2024!"
```

Comment:
```suggestion
    "text": "Influence MUI's roadmap by <a style=\"color: inherit;\" data-ga-event-category=\"Announcement\" data-ga-event-action=\"notification\" data-ga-event-label=\"mui-survey\" href=\"https://tally.so/r/3Ex4PN?source=docs-notification\">answering the 2023 developer survey!</a>"
```

Maintaining word consistency with the other banners and increasing the link size so it's easier to click

Reply:
If we match with https://github.com/mui/material-ui/pull/40553#pullrequestreview-1820109936

```suggestion
    "text": "Influence 2024's roadmap! Take a few minutes to answer the <a style=\"color: inherit;\" data-ga-event-category=\"Announcement\" data-ga-event-action=\"notification\" data-ga-event-label=\"mui-survey\" href=\"https://tally.so/r/3Ex4PN?source=docs-notification\">Developer Survey</a>."
```


<img src="https://github.com/mui/material-ui/assets/3165635/dac7d7fe-44fa-43ba-b306-153ed32f3854" width="369">